libs/@hashintel/type-editor/src/entity-type-editor/link-list-card/link-entity-type-selector.tsx

```diff
@@ -75,7 +75,9 @@ export const LinkEntityTypeSelector = ({
 
   const entityTypeOptions = useFilterTypeOptions({
     typeOptions: entityTypesArray,
-    typesToExclude: chosenEntityTypes,
+    // we pass the selected values to MUI, and can let it identify which are already selected
+    // the isOptionEqualToValue function checks equality
```

Comment:
it's commenting on a value provided to an argument but I prefer JSDoc format for multiline things anyway as it makes it clearer they belong together (rather than stacked unrelated comments)

https://github.com/hashintel/hash/pull/2196/commits/c6cece2f3e7aae34a61f8a75d6957b7a490c72c1